src/clr-angular/accordion/accordion-panel.html

```diff
@@ -11,12 +10,17 @@
         class="clr-accordion-header-button"
         (click)="togglePanel()"
         [id]="getAccordionHeaderId(panel.templateId)"
-        [disabled]="panel.disabled"
+        [disabled]="isAccordion && panel.disabled"
+        [attr.aria-disabled]="!isAccordion && panel.disabled"
```

Comment:
Can you tell me why accordion uses `disabled` attribute and stepper uses `aria-disabled`? Also doesn't presence of `disabled` attribute make it disabled by default even if its value is set to `false`?

Reply:
The reason the accordion uses `disabled` is we want the button truly disabled, cant be focused or clicked. When using `disabled` you can't trigger a focus so on the stepper we use `aria-disabled` so we can auto focus to the next step and read both the step title and let the screen user know the button is disabled.

When `panel.disabled` is `false` that will set the `[disabled]` JS property on the button which will reflect that state into the HTML and remove the `disabled` attribute. If I had used `[attr.disabled]="panel.disabled"` then I set the attribute skipping the property completely and `disabled="false"` would be added to the button disabling it incorrectly.

Reply:
Thanks for the detailed explanation. It all makes sense now. 👍